lib/laa_crime_schemas/types/types.rb

```diff
@@ -188,7 +195,9 @@ module Types
 
     OwnershipType = String.default('applicant').enum(*%w[applicant applicant_and_partner partner])
 
-    EmploymentType = String.enum(*%w[employed self-employed business_partnership director shareholder not_working])
+    EmploymentType = String.enum(*%w[employed self_employed business_partnership director shareholder not_working])
```

Comment:
I think you can remove `business_partnership`, `director` and `shareholder` actually, they are no longer present in the Crime Apply employment status types :)